packages/server/src/rating/__tests__/rating.test.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  getGlickoPlayer,
+  userRankingFromGlickoPlayer,
+  supportsRatings,
+  getGlickoResult,
+} from "../rating";
+import { Glicko2 } from "glicko2";
+import { UserRanking, UserResponse } from "@ogfcommunity/variants-shared";
+
+test("supportsRatings", () => {
+  const supportsRating = supportsRatings("chess");
+  expect(supportsRating).toEqual(false);
+
+  const new_supportsRating = supportsRatings("quantum");
+  expect(new_supportsRating).toEqual(true);
+});
+
+test("getGlickoPlayer - with typical player value", () => {
+  const db_player = {
+    id: "672844d2254d76r4387b8488",
+    login_type: "persistent",
+    username: "testUser",
+    ranking: {
+      quantum: {
+        rating: 1337,
+        rd: 290,
+        vol: 0.0599,
+      },
+      baduk: {
+        rating: 1623,
+        rd: 290,
+        vol: 0.0599,
+      },
+      phantom: {
+        rating: 1762,
+        rd: 290,
+        vol: 0.0599,
+      },
+    },
+  };
+
+  const db_player_ranking_quantum = db_player.ranking["quantum"];
+
+  const ranking = new Glicko2({
+    tau: 0.5,
+    rating: 1500,
+    rd: 350,
+    vol: 0.06,
+  });
+
+  const new_ranking = new Glicko2({
```

Comment:
I believe we do. The Glicko object has some internal variables like `id` that change when we use the object. Since the `.toEqual` function is doing a deep equality, it's comparing all the values in the object including `id`. So, if we didn't have the new rating object the test would fail even though we only want to compare the rating, rd, and vol. Instead we could use the function `toMatchObject` instead of `toEqual` and only compare the properties that we need to compare.